tests/bson/bson-utcdatetime-001.phpt

```diff
@@ -12,21 +12,21 @@ $manager = new MongoDB\Driver\Manager(STANDALONE);
 
 $classname = BSON_NAMESPACE . "\\UTCDateTime";
 $utcdatetime = new $classname("1416445411987");
-$result = $manager->executeInsert(NS, array('_id' => 1, 'x' => $utcdatetime));
-$query = new MongoDB\Driver\Query(array('_id' => 1));
-$cursor = $manager->executeQuery(NS, $query);
-$array = iterator_to_array($cursor);
 
+$manager->executeInsert(NS, array('_id' => 1, 'x' => $utcdatetime));
 
-$date = $utcdatetime->toDateTime();
-var_dump($date->format(DATE_RSS));
+$query = new MongoDB\Driver\Query(array('_id' => 1));
+$cursor = $manager->executeQuery(NS, $query);
+$results = iterator_to_array($cursor);
 
-echo $utcdatetime, "\n";
+$datetime = $utcdatetime->toDateTime();
+var_dump($datetime->format(DATE_RSS));
+var_dump((string) $utcdatetime);
 
 $tests = array(
     array($utcdatetime),
-    array($array[0]->x),
-    array($date),
+    array($results[0]->x),
+    array($datetime),
```

Comment:
This really wasn't intentional.

I would remove this check from the test, and make a totally new testcase for the exact ticket.